commands/version.go

```diff
@@ -41,9 +49,62 @@ func runVersion(cmd *cobra.Command, args []string) {
 		fmt.Println(version.BuildVersion())
 	} else {
 		printFiglet()
-		fmt.Printf("Commit: %s\n", version.GitCommit)
-		fmt.Printf("Version: %s\n", version.BuildVersion())
+		fmt.Printf("CLI commit: %s\n", version.GitCommit)
+		fmt.Printf("CLI version: %s\n", version.BuildVersion())
+		printServerVersions()
 	}
+
+}
+
+func printServerVersions() {
+
+	var services stack.Services
+	var gatewayAddress string
+	var yamlGateway string
+	if len(yamlFile) > 0 {
+		parsedServices, err := stack.ParseYAMLFile(yamlFile, regex, filter)
+		if err == nil && parsedServices != nil {
+			services = *parsedServices
+			yamlGateway = services.Provider.GatewayURL
+		}
+	}
+
+	gatewayAddress = getGatewayURL(gateway, defaultGateway, yamlGateway, os.Getenv(openFaaSURLEnvironment))
+
+	timeout := 2 * time.Second
+	client := proxy.MakeHTTPClient(&timeout)
+	response, err := client.Get(gatewayAddress + "/system/info")
```

Comment:
instead of `client.Get`, please use ` request,_ := http.NewRequest` and `client.Do(request)`

Reply:
isn't that exactly what `client.Get` does?

Reply:
Please change for consistency.